CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -1087,7 +1087,7 @@ public static function processRecurringContribution(&$form, &$params, $contactID
     $recurParams['invoice_id'] = $params['invoiceID'] ?? NULL;
     $recurParams['contribution_status_id'] = CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Pending');
     $recurParams['payment_processor_id'] = $params['payment_processor_id'] ?? NULL;
-    $recurParams['is_email_receipt'] = $params['is_email_receipt'] ?? NULL;
+    $recurParams['is_email_receipt'] = $params['is_email_receipt'] ?? '';
```

Comment:
It should be the case that a BOOL works for boolean fields - so 
```
 $recurParams['is_email_receipt'] = (bool) ($params['is_email_receipt'] ?? FALSE);
```

should work